stl/inc/yvals_core.h

```diff
@@ -2014,7 +2014,9 @@ compiler option, or define _ALLOW_RTCc_IN_STL to suppress this error.
 #define _STL_INTERNAL_STATIC_ASSERT(...)
 #endif // ^^^ !defined(_ENABLE_STL_INTERNAL_CHECK) ^^^
 
-#ifdef __CUDACC__ // TRANSITION, CUDA 12.4 doesn't have downlevel support for static call operators
+#if defined(__CUDACC__) || (defined(__clang__) && __clang_major__ < 16)
+// TRANSITION, CUDA 12.4 doesn't have downlevel support for static call operators
+// TRANSITION, VSO-2397560, we don't generally attempt to support old versions of Clang

```

Comment:
non-blocking nit - "we don't generally attempt to support old versions of Clang" sounds to me like we've decided to make a (possibly permanent) exception.

I suggest to be more explicit on the intent to remove this, as below.

```suggestion
// TRANSITION, VSO-2397560, _temporary_ workaround for RWC relying on ancient Clang versions.
```

Reply:
Yeah, that's much better, since I don't want to give anyone justification for complaining when we eventually remove this. Thanks!